sync-mangle.js

```diff
@@ -0,0 +1,43 @@
+const { resolve } = require('path');
```

Comment:
Perhaps we should add this file to a `scripts` directory?

Reply:
I like it, I'll add that